### PR TITLE
Convert top menu to Bootstrap's navbar component

### PR DESCRIPTION
**Pull request by MauricioFauth**

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
@sudo-bot deploy PR

File: templates/top_menu.twig [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        {% for tab in tabs %}
+          <li class="nav-item{{ tab.active ? ' active' }}">
+            <a class="nav-link" href="{{ url(tab.route, url_params|merge(tab.args ?? [])) }}">
+              {{ get_icon(tab.icon, tab.text, false, true, 'TabsMode') }}
+              {% if tab.active %}
+                <span class="sr-only">{% trans %}(current){% notes %}Current page{% endtrans %}</span>
```
Comment (williamdes) [inline templates/top_menu.twig]: ref #13220 ?

**Review by williamdes — APPROVED** [review]
image
I know this is a draft but on small screen the tabs are hidden

**Comment by mauriciofauth** [comment]
> I know this is a draft but on small screen the tabs are hidden

There is a hidden button on the left side that toggles the menu. Also the 'more' dropdown still not works.

File: templates/top_menu.twig (hunk 1 above)
Comment (mauriciofauth) [reply templates/top_menu.twig]: Not intentional. I just followed the [Navbar component example](https://getbootstrap.com/docs/4.4/components/navbar/).

**Comment by williamdes** [comment]
@sudo-bot /deploy PR

**Review by williamdes — APPROVED** [review]
image

Usually the menu "burger" button is on the right
Can you make the bars black ?

**Comment by williamdes** [comment]
Can you please fix the effect ?
screencast-phpmyadmin-pr-15775 wdes eu org-2020 01

**Comment by mauriciofauth** [comment]
> Usually the menu "burger" button is on the right
Can you make the bars black ?

The toggle button is left aligned by default and the icon is using the same color as the links also by default.

> Can you please fix the effect ?

I disabled the transitions altogether. I don't know what is causing this. I need to investigate it further.

**Comment by williamdes** [comment]
Closes: #14183

**Pull request merged**